release-notes.txt

```diff
@@ -3544,7 +3544,7 @@ Bootloader
 -----------
 + MCUBoot: A secure bootloader for ARM MCUs is now supported. This initial support
 includes only the nrf52dk board. For more information about it refer to the mcuboot github
-page at https://github.com/runtimeco/mcuboot/blob/master/README-RIOT.rst
+page at https://github.com/JuulLabs-OSS/mcuboot/blob/master/docs/readme-riot.md
```

Comment:
I'm not really sure if we should modify the release notes actually...

Reply:
Me too.

Reply:
If we see this as an archive, then I would revert the change here.

Reply:
Reverted.